pages/index.js

```diff
@@ -116,6 +117,31 @@ const GraphNumber = s(Title)`
   font-size: 50px;
 `
 
+const GraphNumberBubble = s.div`
+  line-height: 1;
+  transform: translate(0, 20%);
+  color: #D12D4A;
```

Comment:
@peterbaile make a decision on whether to use hex code or rgb and standardize across the repo?

Reply:
there's rgb operator in line 94 but also hexcode